src/app/home/page.tsx

```
@@ -19,18 +19,181 @@ const monthNames = [
+
+    switch (type) {
+      case "ongoing":
+        cardClasses =
+          "bg-gradient-to-r from-orange-400 to-orange-500 text-white border-orange-300";
+        statusText = `진행중 (${formatDate(challenge.endDate)}까지)`;
+        statusColor = "bg-orange-600";
+        break;
+      case "upcoming":
+        const daysLeft = getDaysUntilStart(challenge.startDate);
+        cardClasses =
+          "bg-gradient-to-r from-blue-400 to-blue-500 text-white border-blue-300";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)